.changeset/calm-eyes-return.md

```diff
@@ -0,0 +1,6 @@
+---
+"docs": patch
+"@espressive/cascara": patch
+---
+
+doc(FDS-377): [Docs] use Cascara's AdminStructure in Docs
```

Comment:
@lordpixel @rociobaena just a heads up... we do not want to include changesets for Docs. The reason for this is that Docs and Cascara are linked in our Changeset config. This means that when the Cascara version changes, it syncs the version of Docs to keep in line with it. If we do a version bump of the docs, we will actually also bump the version of Cascara inadvertently.

We should consider a docs update similar to any other internal tooling, which we do not want to version bump as those versions are not for us internally, but instead they are used for anyone external to our project to know what we have changed.

Reply:
Makes sense, removed, thanks for the heads up 👍